.github/workflows/ci.yml

```diff
@@ -23,6 +26,10 @@ jobs:
          ghc: ["8.10.7", "9.6.4", "9.8.1"]
          cabal: ["3.10.2.1"]
          os: [ubuntu-latest]
+         variant: [default, no-thunks]
+         exclude:
+          variant:
+            ${{ github.event_name == 'schedule' ? 'no-thunks' : 'default' }}
```

Comment:
fixed `exclude` not containing a list 

I'm not sure I understand what you mean with the second part, though. As I understand it at the moment, it will always remove one or the other, so we're either excluding `no-thunks` in non-nightly runs, or `default` the nightly runs.

Reply:
ah, sorry, you're right, the conditional is flipped, I'll fix it